src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import Icon from 'supercons'
+
+export default function SearchBar({ userInput, setUserInput }) {
+  return (
+    <InputLabel>
+      <Icon glyph="search" size={30} />
+      <Input
+        value={userInput}
+        onChange={event => setUserInput(event.target.value)}
+        maxLength={200}
+        placeholder="Search books"
```

Comment:
I hope I haven't misread it, but I've seen in a filter on the BookCardsPage, that your search accepts both author's names and book titles. You could let your users know via the placeholder text, that they can search for either here.

Reply:
Great idea, ill do so. Thanks!